packages/sw-background-sync-queue/src/index.js

```diff
@@ -12,7 +12,10 @@
  */
 
 /**
- * sw-background-sync-queue Module
+ * # sw-background-sync-queue
+ *
+ * A service worker implementation of the a queue which is triggered by
```

Comment:
Reading this description I'm not convinced it's the best way to describe the module.

Would something along the lines of:

"Queues failed requests and uses the Background Sync API to replay those requests at a later time when the network state has changed."